### ISSUE TITLE
about suggestions

### ISSUE THREAD

**Issue by skyhhh666** [drojdjou/J3D#23]
Hello, I saw your project Unity3d to Three.js exporter on github. Thank you for sharing. We have always wanted to develop a plug-in like this. After you said in it, you may export: textures, lights, uv (multiple channels), lightmaps, more materials, any camera type, multiple cameras. But I didn't see you continue, is there any technical difficulties? Because we also want to develop such a plug-in, see if we can get some technical guidance and suggestions from you, thank you! Looking forward to your reply.

**Comment by drojdjou** [comment]
> is there any technical difficulties?

No, I just abandoned the project because I started another job and was no longer dealing with Unity3D and Three.js, so there was no time or motivation for me to continue. I think that Unity has a decent WebGL exporter at this point, but this kind of project can still make sense if you would like to work on it

**Comment by 1170300814** [comment]
> > 有什么技术困难吗？
> 
> 不，我只是放弃了这个项目，因为我开始了另一份工作，不再处理 Unity3D 和 Three.js，所以我没有时间或动力继续下去。我认为 Unity 目前有一个不错的 WebGL 导出器，但如果您想从事它的工作，这种项目仍然有意义

which one? please tell me
